src/components/Nav/Nav.js

```diff
@@ -1,9 +1,53 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import { CiSearch } from 'react-icons/ci';
+import { BsCart3 } from 'react-icons/bs';
+import { HiOutlineBars3 } from 'react-icons/hi2';
+import { LINK_LIST, LINKBTM_LIST } from './NavData.js';
+import './Nav.scss';
 
 const Nav = () => {
   return (
-    <div>
-      <h1>nav</h1>
+    <div className="nav">
+      <div className="navTop">
+        <div className="memberLink">
+          {LINK_LIST.map(list => {
+            return (
+              <Link className="memberLinkStyle" to={list.to} key={list}>
+                {list.title}
+              </Link>
+            );
+          })}
+        </div>
+        <div className="navMain">
+          <Link to="/" className="navLogo">
+            SIMPLE
+          </Link>
+          <div className="navSearch">
+            <input className="navInput" />
+            <button className="navButton">
+              <CiSearch className="navButtonIcon" />
+            </button>
+          </div>
+          <BsCart3 className="navCart" />
+        </div>
+      </div>
+      <div className="navBtm">
```

Comment:
```suggestion
      <div className="navBtn">
```
혹시 Btm이 button의 약자로 쓴거라면 오타가 있네요!

Reply:
피드백 감사합니다 멘토님!!
지금 컴포넌트에서 클라스를 크게 navTop / navBtm (bottom) 으로 두개 나누었는데 Bottom으로 수정하는게 좋을까요?

Reply:
아 그런 의미라면 className 작성에서 의미 있게 작성하는 것이 좋겠습니다.
navTop과 navBottom은 Nav에 위와 아래라는 의미만 있지 어떤 역할을 하고 있는 영역이라는 것을 드러내지 못합니다.

그래서 navTop 이 부분은 Link들을 담고 있는 영역이기에 linkBox 혹은 linkWrap 이 더 어울리는 이름이겠네요!
이 부분 참고하셔서 다른 className도 고민하시고 변경해 주시면 좋겠습니다!

Reply:
넵 감사합니다!